__tests__/api.test.js

```diff
@@ -82,6 +82,14 @@ describe("Full API Test Suite", () => {
           expect(msg).toBe("Not found");
         });
     });
+    test("200: Feature test, adding comment_count to the returned article", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article[0]).toHaveProperty("comment_count");
```

Comment:
I would check how many comments are there! 

Also, because you are expecting 1 article back you shouldn't have an array to deal with. return rows[0] in the model and then you just have an object to deal with!